Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -324,13 +324,6 @@ void RenderSVGText::layout()
     // textElement().updateLengthContext();
 
     bool updateCachedBoundariesInParents = false;
-    if (!isLayerBasedSVGEngineEnabled()) {
-        if (m_needsTransformUpdate) {
-            m_localTransform = textElement().animatedLocalTransform();
```

Comment:
Hmm, I naively expected a side-effect here, as SVG text layout code does access the 'm_localTransform' during layout, when computing the on-screen font-size scaling factor (which cries for a less involved replacment, but that's another patch...).

if the 'computeNewScaledFont...' code runs during layout(), you're using the SVG <text> 'transform' information from the _previous_ layout run (when it was last updated at the end of the last RenderSVGText::layout() cycle). Also first layout() misses the transform from the \<text\>, if specified.

So \<text transform="scale(10)" font-size="8px"/\> should be affected by this. If not, try a dynamic change of setting transform to scale(10) after the document loaded / painted. Please check if that is affected, to be on the safe side.

I'm asking as LBSE currently has issues with <text> + transform (latter is ignored), did not check the cause yet - still busy with compositing related work, before tackling bug fixes / features in LBSE.

Reply:
Note: in the other resolved commented you stated "It also looks like supplementalTransform() is the only code in here that could read layout information" -- that is correct and it is important to check who can read layout information, but it's equally important to check if 'm_localTransform' is actually accessed between the old location and the new one (where you moved the code to, below in the layout() function)).

Reply:
I ran the suggested commands locally, the only changes found were the ones caught by EWS, the remainder was mostly noise, or flaky screenshotting.